mmit/core/piecewise_function.cpp

```diff
@@ -146,7 +146,7 @@ bool has_slack_at_position(double func_breakpoint_pos, double func_sign, Coeffic
     }
 }
 
-int PiecewiseFunction::insert_point(double y, bool is_upper_bound) {
+int PiecewiseFunction::insert_point(double b, Coefficients F, bool is_upper_bound) {
```

Comment:
Good idea, but I'm not sure we can do that. For the linear hinge loss, we wouldn't need this argument. However, for the squared hinge loss, the coefficients specify a parabola and we can't detect which type of limit if corresponds to. We need this info to know on which side of the breakpoint the function is worth 0. Any ideas?